### DISCUSSION TITLE
it says user not authorized.

### DISCUSSION THREAD

**Discussion by TarunavBA** [community/community#8187] [Copilot Conversations] [answered]
it says user not authorized, but it didn't prompt me for authorizing. Now the icon shows but it says user not authorized. I have disabled the HTTP proxy support for the extension also. Is there a possible workaround for this?
Here is a screenshot:-
![image](https://user-images.githubusercontent.com/89698728/143733368-cf1019e8-ff26-4527-b7c8-fd8b5c7ede4d.png)

**Comment by TarunavBA** [comment, marked as answer]
hey @davecheney Do you have a workaround?

**Reply by davecheney** [reply]
I’m sorry you are still having problems. From all the information I have available to me the best I can suggest is your computer is firewalled from the internet.

**Reply by avigloz** [reply]
I'm having the same issue. It's not a firewall issue for me, I'm certain of that.
![image](https://user-images.githubusercontent.com/5449202/143940736-0918f878-e696-447d-87b5-32d43c51dae6.png)

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by ishivamshukla** [comment]
You're still on the waitlist. 
and wait we should use it after receiving the official mail.
